xrootd/streammanager/streammanager.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2018 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package streammanager // import "go-hep.org/x/hep/xrootd/streammanager"
```

Comment:
I am not too fond about the name of the package.

trying to come up with a (IMO) better name... basically, the main type of that package "routes" server responses, right?
something akin to [Gorilla's router/multiplexer](https://godoc.org/github.com/gorilla/mux) ?

I'd be tempted to rename `streammanager` into `mux`, `router` or `dispatch`.
what do you think?